src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -170,23 +131,9 @@ void GcsResourceManager::HandleGetAllAvailableResources(
     rpc::AvailableResources resource;
     resource.set_node_id(node_id.Binary());
 
-    for (size_t i = 0; i < node_resources.predefined_resources.size(); ++i) {
-      const auto &resource_value = node_resources.predefined_resources[i].available;
-      if (resource_value <= 0) {
-        continue;
-      }
-
-      const auto &resource_name = scheduling::ResourceID(i).Binary();
-      resource.mutable_resources_available()->insert(
-          {resource_name, resource_value.Double()});
-    }
-    for (const auto &entry : node_resources.custom_resources) {
-      const auto &resource_value = entry.second.available;
-      if (resource_value <= 0) {
-        continue;
-      }
-
-      const auto &resource_name = scheduling::ResourceID(entry.first).Binary();
+    for (const auto &resource_id : node_resources.available.ResourceIds()) {
```

Comment:
`available` can be negative and the old code filters it out. We should do the same thing here?